conbench/app/_plots.py

```diff
@@ -18,6 +19,13 @@
 log = logging.getLogger(__name__)
 
 
+@dataclasses.dataclass
+class BokehPlotJSONOrError:
+    # mutually exclusive
+    jsondoc: Optional[str]
+    reason_why_no_plot: Optional[str]
```

Comment:
tiny types like this are not set in stone, can and should always be iterated upon -- but they really really help navigating code quickly :)